src/main/java/eu/miaplatform/customplugin/ServiceClientFactory.java

```diff
@@ -17,4 +22,12 @@ public static CRUDServiceClient getCRUDServiceClient(String apiPath, int version
     public static CRUDServiceClient getCRUDServiceClient(String apiPath, String apiSecret, int version, CustomPluginHeadersPropagator headersPropagator) {
         return new CRUDServiceClientImpl(apiPath, apiSecret, version, headersPropagator);
     }
+
+    public static Service getDirectServiceProxy(String serviceName, InitServiceOptions options) {
+        return new Service(serviceName, options);
+    }
+
+    public static Service getServiceProxy(InitServiceOptions options) {
+        return new Service(System.getenv(MICROSERVICE_GATEWAY_SERVICE_NAME_KEY), options);
```

Comment:
We should throw an exeption if env is not set: `System.getenv(MICROSERVICE_GATEWAY_SERVICE_NAME_KEY)`.

The best solution would be to check required env variables as soon as possible during service setup. 

We could define a new class that is responsbile of parsing the environment, using a static method such as `EnvManager.getEnvVariables()`.

In the future this method could also receive a map of environment variables to get, this way the user of the library can define its own env vars and access them through our class.